src/mscorlib/shared/System/IO/Path.cs

```diff
@@ -477,6 +477,22 @@ public static bool TryJoin(ReadOnlySpan<char> path1, ReadOnlySpan<char> path2, R
             return true;
         }
 
+        public static bool TryGetTempPath(Span<char> destination, out int charsWritten)
+        {                        
+            charsWritten = 0;
+
+            var tmpPath = Path.GetTempPath();
+
+            if (tmpPath.Length > destination.Length)
+                return false;            
+
+            tmpPath.AsSpan().CopyTo(destination);
+
```

Comment:
nit: extraLine

Reply:
@Anipik is there a guideline for "extraLine"?